src/framework/preferences/css/sass/utils/_fonts.scss

```diff
@@ -2,6 +2,7 @@
 
 $fonts: (
     ".fl-font-arial": "Arial",
+    ".fl-font-system-ui": "system-ui",
```

Comment:
To avoid confusing ourselves in future, should we rename all these uses of "system" to "device" also?

Reply:
Good question. The actual generic font-family is called [system-ui](https://w3c.github.io/csswg-drafts/css-fonts-4/#system-ui-def). So the right hand side can't be changed. We could change the name of the class to device if that's preferred. I'm fine either way, I guess we need to decide if we want the class name to reflect the font family or the name exposed via the adjuster.